### DISCUSSION TITLE
Image Cover & Logo - As Notion

### DISCUSSION THREAD

**Discussion by CyRamos** [anpigon/obsidian-book-search-plugin#53] [Q&A]
Hey :)
I noticed your reading's note page and it looks exactly like Notion pages! [Attached image for example]
1. I'd like to know which plugin are you using to organize logos and covers.
2. I also like to figure out how did you order your "read" / "reading" / "unread" headlines like this on the left and many linked notes in the table itself - is this another plugin or the original dataview plugin? 

Many thanks ahead 👍 

![image](https://user-images.githubusercontent.com/40651473/197421690-faf92872-1cfe-45ef-9929-c5b5d6630d44.png)

**Comment by IDsetters** [comment]
Hi, I love this plugin. I wanted to insert other books from the same author. I tried this but didn't work. 
![image](https://github.com/user-attachments/assets/efb775bd-9b67-49cf-bbd4-9c2024fb6819)

```dataview
table title as "Book Title", file.link as "Link"
from "Books"
where contains(tags, "#author/John_Doe")
sort publish_date desc```

Any one has a suggestion? Thank you so much
